docs/post_installation/post_installation.md

```diff
@@ -0,0 +1,166 @@
+# Post-installation steps
+Once the Hornet is deployed, all parameters are set via configuration files.
+
+## Configuration
+The most important ones are:
+* `config.json`: include all configuration flags and their values
+* `peering.json`: include all connection details to your static peers (neighbors)
+
+Since the Hornet node software is able to power original IOTA 1.0 network as well as IOTA 1.5 (aka Chrysalis), it is important to use respective `config.json` file that targets the IOTA network that we want. All configuration files that targets respective networks are available in [source code repo](https://github.com/gohornet/hornet/tree/master) on GitHub.
```

Comment:
That's not correct, there are distinct versions of Hornet which are compatible to the different protocols but there isn't a version which supports all. The configuration for these versions are completely different.

Reply:
Great catch. Thank you.

Reply:
Changed